lesson04/assignment/css/_menu_animation.scss

```diff
@@ -0,0 +1,71 @@
+// === Variables =====
+$perspective:     60rem;
+$font-size:       1.25rem;
+$split-position:  49%;
+$split-thickness: 4px;
+$split-color:     black;
+
+// === Menu <ul> =====
+.Menu-list {
```

Comment:
Remember that naming conventions in HTML and CSS are "kebab case"- all letters are lower case, and spaces are replaced with hyphens. So this class name should be "menu-list".